cpg_infra/billing_aggregator/aggregate/utils.py

```diff
@@ -671,8 +673,10 @@ async def _get_jobs_and_add_to_queue(
 
         batch_name = batch.get('attributes', {}).get('name')
         # check if hail batch query and too many jobs
-        if batch_name is None and jobs_cnt > DEFAULT_MAX_JOBS_PER_BATCH:
-            # This is most likely Hail Batch Query job, aggregate it as one job
+        if (batch_name is None and jobs_cnt > SEQR_MAX_JOBS_PER_BATCH) or (
+            batch_name is not None and jobs_cnt > HAIL_MAX_JOBS_PER_BATCH
```

Comment:
I think this is right, but the names of these variables are confusing, can you align them closer to what they represent, eg: `HAIL_QUERY_JOB_PER_BATCH_LIMIT` and `HAIL_NON_QUERY_JOB_PER_BATCH_LIMIT`?